### PR TITLE
Double quotes is added for a Word since it was Confusing without the Quotes

### PR DESCRIPTION
**Pull request by law27**
Alphabet word is not wrapped inside double quotes

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Comment by moT01** [comment]
I have blocked this for now. See the [pinned code freeze issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/38308) - The PR can still be and reviewed, just not merged at the moment.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 , once the _merge-block_ is lifted

**Review by moT01 — APPROVED** [review]
Congratulations on your first contribution to freeCodeCamp @law27 🎉

**Pull request merged**
